devops/README.md

```diff
@@ -5,9 +5,14 @@
 
 # Hands-on Task
 
-- Pick one of Java, Python or Docker Container type of an artifact that is a typical simple web application. You can pick any open-source project artifact or a simple helloworld container.
+- Pick one of Java, Python or Docker Container type of an artifact that is a typical simple web application. You can pick any open-source project artifact or a simple container you develop.
+  
+- Write a CI / CD pipeline for the artifact, using the CI tool of your choice.
+  
+- Implement 2 security testing tools of your choice in the CI/CD pipeline
+
+- Write simple deployment automation script that deploys the given artifact on AWS.
```

Comment:
Not sure if we want this, the CI\CD pipeline should deploy it to AWS, they can use a script, but I read it as needing both a script and a CI\CD pipeline where we just need a CI\CD pipeline I think.